frontend/third_party/argo-ui/argo_template.ts

```diff
@@ -1076,7 +1076,7 @@ export interface DAGTask {
    */
   dependencies: string[];
 
-  // TODO: This exists in https://github.com/argoproj/argo/blob/master/api/openapi-spec/swagger.json
+  // TODO: This exists in https://github.com/argoproj/argo-workflows/blob/master/api/openapi-spec/swagger.json
   // but not in https://github.com/argoproj/argo-ui/blob/master/src/models/workflows.ts
```

Comment:
nit: this link also seems to be broken but not sure what it should point to since the `workflows.ts` files seems to be removed.

Reply:
This file appears to have been moved into the argo-workflows repo. Updated the link.